packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -497,21 +498,21 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
   }
 
   // Logic for updating `fieldTranslations`
-  // user has no description translations OR user is changing the description
-  const descriptionTranslationsNeeded =
+  // user has no translations OR user is changing the field
+  const hasNoDescriptionTranslations =
     eventType.fieldTranslations.filter((trans) => trans.field === EventTypeAutoTranslatedField.DESCRIPTION)
-      .length === 0 || newDescription;
-  const description = newDescription ?? eventType.description;
-
-  if (
-    ctx.user.organizationId &&
-    autoTranslateDescriptionEnabled &&
-    descriptionTranslationsNeeded &&
-    description
-  ) {
-    await tasker.create("translateEventTypeDescription", {
+      .length === 0;
+  const description = newDescription ?? (hasNoDescriptionTranslations ? eventType.description : undefined);
```

Comment:
newDescription is undefined unless user modifies the description of the event-type